.github/workflows/plugin_tests.yaml

```diff
@@ -0,0 +1,46 @@
+name: Remote Plugin Tests
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    timeout-minutes: 45
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, windows-latest, macos-latest]
```

Comment:
I think it's okay to run only on `ubuntu-20.04`/`ubuntu-latest`.

Reply:
Yeah I originally had it as only running on ubuntu, but I was thinking that with some remote types we do get platform specific credential issues (mainly w/ssh). But I guess since we are only testing s3 for now it's probably safe to only test linux